### PR TITLE
poster fix

### PR DESCRIPTION
**Pull request by Nickpopadopolous**

### CODE REVIEW

File: assets/javascript/java1.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ $(document).ready(function () {
 		console.log(queryURL);
 
 		$.ajax(request).done(function(response) {
-
+console.log(response);
```
Comment (dzimmerman902) [inline assets/javascript/java1.js]: Did you add this response? If you don't need it, feel free to delete it.

**Comment by dzimmerman902** [comment]
I noticed a console.log was added, do we need that?

**Comment by Nickpopadopolous** [comment]
Oh whoops no!

File: secondPage.html [hunk 2]
```diff
@@ -180,7 +180,7 @@ <h5 id="movieName"></h5>
         </div>
 
 		<div class="tabs-panel" id="panel3">
-
+	<h4>If you liked that movie, here are some related titles for you!</h4>
```
Comment (dzimmerman902) [inline secondPage.html]: Update the text a little bit, technically they haven't seen the movie yet, maybe make it say something like "Fans who liked the movie...."

**Pull request merged**